frontend/app/src/components/helper/ExternalLink.vue

```diff
@@ -1,12 +1,69 @@
-<script lang="ts" setup>
-const props = defineProps<{
-  url: string;
-}>();
+<script setup lang="ts">
+const props = withDefaults(
+  defineProps<{
+    url?: string | null;
```

Comment:
```suggestion
    url?: string;
```

and make the default `undefined`. In the past we needed null because undefined was not triggering reactivity after being set, but this has not be a problem with composition api.